src/cutadapt/expected_errors.h

```diff
@@ -0,0 +1,163 @@
+#include <stdint.h>
+#ifdef __SSE2__
+#include "emmintrin.h"
+#endif
+
+static const float SCORE_TO_ERROR_RATE[94] = {
+    1.0L,                     // 0
```

Comment:
Because it was originally `static const double`, but when I started vectorizing I realized it was a lot more tedious to have 2 numbers per vector as opposed to 4 numbers using floats. It does not really matter for the result. If you want I can remove the Ls.

Reply:
Ok, just leave it as it is.